reporters/default_reporter_test.go

```diff
@@ -109,6 +109,12 @@ func S(options ...interface{}) types.SpecReport {
 			report.AdditionalFailures = append(report.AdditionalFailures, option.(types.AdditionalFailure))
 		case reflect.TypeOf(0):
 			report.NumAttempts = option.(int)
+		case reflect.TypeOf(true):
```

Comment:
I think, instead, we can have a new `type MaxFlakeAttempts int` and `type RequiredPasses int`

Reply:
Should NumAttempts be removed then?
If we add these new type, then one of them will always match NumAttempts.

Reply:
Yes I think that's right.

Given the phrasing I think the defaults would be `MaxFlakeAttempts = 0` and `RequiredPasses = 1` right? If we want the defaults to be zero it could be `RequiredAdditionalPasses`